packages/core/src/utils/react-query-utils.ts

```diff
@@ -146,7 +147,27 @@ export const retryFunction = (failureCount: number, error: any) => {
   return false
 }
 
-export const defaultQueryConfig = {
-  suspense: true,
-  retry: retryFunction,
+export function getDefaultQueryConfig() {
```

Comment:
This is probably a premature optimization, but I wonder if this function should be memo-ized.  I don't think its return value can change after it has been run once.

Reply:
`getConfig` is already memoized and reading from `process.env` should be quick